### PR TITLE
created initial season endpoint

### PR DESCRIPTION
**Pull request by kevanhannah**
Created the season endpoint and corrected a typo!

### CODE REVIEW

File: router/season.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        Season.find((err, season) => {
+            if (err) reject(err);
+            resolve(season);
+        });
+    })
+    .then(season => {
+        res.json(season);
+    })
+    .catch(err =>
+        res.json(err));
```
Comment (GarrettLevine) [inline router/season.js]: you're missing sone `{ }` here around the `.catch` anon function and close `)`

File: models/Episode.js [hunk 2]
```diff
@@ -30,7 +30,7 @@ const EpisodeSchema = new Schema({
     winner: [labelIdType],
   },
   bottomTwo: [labelIdType],
-  lipSynchSong: {
+  lipsyncSong: {
```
Comment (GarrettLevine) [inline models/Episode.js]: 😅

File: models/Season.js [hunk 3]
```diff
@@ -8,8 +8,8 @@ const SeasonSchema = new Schema({
     required: true,
   },
   queens: [labelIdType],
-  airDate: {
-    type: Date,
+  year: {
```
Comment (GarrettLevine) [inline models/Season.js]: 👍

**Comment by kevanhannah** [comment]
Adjusted the syntax notedon season.js (and queen.js for consistency) to be single-line arrow functions.

File: router/index.js [hunk 4]
```diff
@@ -3,7 +3,9 @@ const express = require(`express`);
 const router = express.Router();
 
 const queenRouter = require(`./queens`);
+const seasonRouter = require(`./season`);
 
 router.use(`/queens`, queenRouter);
+router.use(`/season`, seasonRouter);
```
Comment (GarrettLevine) [inline router/index.js]: should we also change `queens` to be singular then? probably for consistency.

Comment (GarrettLevine) [inline router/index.js]: @kevanhannah

Comment (kevanhannah) [inline router/index.js]: @GarrettLevine Yes, good idea :+1:

**Pull request merged**